cclib/bridge/__init__.py

```diff
@@ -7,23 +7,13 @@
 
 """Facilities for moving parsed data to other cheminformatic libraries."""
 
-try:
-    import openbabel
-except ImportError:
-    pass
-else:
+from cclib.parser.utils import find_package as _find_package
```

Comment:
Leading underscores are typically used for names internal to the imported module - which is not the case here.

https://www.python.org/dev/peps/pep-0008/#id36